plutus-core/executables/uplc/Main.hs

```diff
@@ -221,7 +303,19 @@ runPrint (PrintOptions inp mode) =
 runConvert :: ConvertOptions -> IO ()
 runConvert (ConvertOptions inp ifmt outp ofmt mode) = do
     program <- (getProgram ifmt inp :: IO (UplcProg PLC.AlexPosn))
-    writeProgram outp ofmt mode program
+    let shrunk = shrinkProgram program
```

Comment:
Just some stuff to make it easy for me to modify files and see how things change.